presenters/thread.rb

```diff
@@ -96,7 +96,7 @@ def get_paged_merged_responses(thread_id, responses, skip, limit, recursive=fals
     paged_response_ids = limit.nil? ? response_ids.drop(skip) : response_ids.drop(skip).take(limit)
     if recursive
       content = Comment.where(comment_thread_id: thread_id).
-        or({:parent_id => {"$in" => paged_response_ids}}, {:id => {"$in" => paged_response_ids}}).
+        any_of({:parent_id => {"$in" => paged_response_ids}}, {:id => {"$in" => paged_response_ids}}).
```

Comment:
The way this API works has changed in mongoid. 

So while the previous version was equivalent to:
`comment_thread_id == thread_id AND (parent_id in paged_response_ids OR id in paged_response_ids)`
It is now equivalent to 
`comment_thread_id == thread_id  OR parent_id in paged_response_ids OR id in paged_response_ids)`
since the where is merged into the or. Using `any_of` fixes this issue ([ref](https://www.mongodb.com/docs/mongoid/current/release-notes/mongoid-7.1/#logical-operations))